test/test-alternates.sh

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env bash
+
+. "test/testlib.sh"
+
+begin_test "alternates (single)"
+(
+  set -e
+
+  reponame="alternates-single-alternate"
+  setup_remote_repo_with_file "$reponame" "a.txt"
+
+  pushd "$TRASHDIR" > /dev/null
+    clone_repo "$reponame" "${reponame}_alternate"
+  popd > /dev/null
+
+  rm -rf .git/lfs/objects
+
+  alternate="$TRASHDIR/${reponame}_alternate/.git/objects"
+  echo "$alternate" > .git/objects/info/alternates
+
+  GIT_TRACE=1 git lfs fetch origin master 2>&1 | tee fetch.log
+  [ "0" -eq "$(grep -c "sending batch of size 1" fetch.log)" ]
```

Comment:
Is there something else we can check in these tests to be more confident that the files were successfully downloaded?

Reply:
Unfortunately not, since what we're testing is that the objects _are_ downloaded, but not via a network request (i.e., they are absent from our repository's .git/lfs/objects, but were able to be found in another alternate).